facia-tool/app/model/frontsapi.scala

```diff
@@ -127,24 +127,19 @@ trait UpdateActions extends Logging with ExecutionContexts {
     FaciaApiIO.putCollectionJson(id, collectionJson)
 
   //Archiving
-  //Archiving
   def archivePublishBlock(collectionId: String, collectionJson: CollectionJson, identity: User): CollectionJson = {
-    FaciaToolArchive.archive(ArchiveRequest(collectionId, identity.email, Json.toJson(collectionJson), Json.obj("action" -> "publish")))
```

Comment:
this is another of those features that stopped working during migration. I'm working on a different way of auditing actions.
`FaciaToolArchive` was writing in dynamodb right?